Jenkinsfile

```diff
@@ -1361,6 +1373,49 @@ pipeline {
                         }
                     } // post
                 } // stage('Functional on Ubuntu 20.04')
+                stage('Test CentOS 7 RPMs') {
+                    when {
+                        beforeAgent true
+                        expression { ! skip_test_rpms_centos7() }
+                    }
+                    agent {
+                        label 'ci_vm1'
+                    }
+                    steps {
+                        testRpm inst_repos: daos_repos(),
+                                daos_pkg_version: daos_packages_version()
+                   }
+                } // stage('Test CentOS 7 RPMs')
+                stage('Scan CentOS 7 RPMs') {
+                    when {
+                        beforeAgent true
+                        expression { ! skip_scan_rpms_centos7() }
+                    }
+                    agent {
+                        label 'ci_vm1'
+                    }
+                    steps {
+                        scanRpms inst_repos: daos_repos(),
+                                 daos_pkg_version: daos_packages_version(),
+                                 inst_rpms: 'clamav clamav-devel',
+                                 test_script: 'ci/rpm/scan_daos.sh',
+                                 junit_files: 'maldetect.xml'
+                    }
+                    post {
+                        always {
+                            junit 'maldetect.xml'
+                        }
+                    }
+                } // stage('Scan CentOS 7 RPMs')
+            } // parallel
+        } // stage('Test')
+        stage('Test Hardware') {
+            when {
+                beforeAgent true
+                expression { ! skip_testing_stage() }
+                expression { ! skip_unstable() }
```

Comment:
You have `skip_unstable()` in `skip_testing_stage()` already.  Moreover the intention is ambiguous.  Do you want this _skip if status is already UNSTABLE_ to apply to the _Test_ stage as well as _Test Hardware_ since you do have  `skip_unstable()` rolled into `skip_testing_stage()` and the latter is applied to both of your newly split stages.

Probably for most clarity each of your new stages needs it's own `skip_*()` and if for the time being they are the same logic, one can just call the other.

But ultimately I think we need clarify the intention of what we want to skip when the status is UNSTABLE.  I am concerned that applying this to the _Test_ stage as has been done is going to trigger it to be skipped when NLT is UNSTABLE (or at least it should, IMO -- that being that if NLT being UNSTABLE doesn't make the job UNSTABLE, I think that's a problem.  It should be UNSTABLE and require force-landing if it really is to be ignored.).

Reply:
oops, I forgot to remove that after moving it.